pkg/kudoctl/cmd/init/manager.go

```diff
@@ -82,6 +83,25 @@ func Install(client *kube.Client, opts Options, crdOnly bool) error {
 	return nil
 }
 
+func ValidateManager(client *kube.Client, opts Options) error {
```

Comment:
since this one is not used or tested, should we just drop it for now? #deadcode

Reply:
Yeah, it's not something so complex that i'd want to keep it around.